dir.props

```diff
@@ -272,6 +272,8 @@
 
     <!-- Constructed shared fx path for testing -->
     <TestSharedFxDir Condition="'$(BinPlaceTestSharedFramework)' == 'true'">$(NETCoreAppTestRootPath)</TestSharedFxDir>
+    <TestSharedFxDir Condition="'$(BuildingUAPAOTVertical)' == 'true'">$(ILCFXInputFolder)</TestSharedFxDir>
+    <UseDotNetNativeToolchain Condition="'$(BuildingUAPAOTVertical)' == 'true'">true</UseDotNetNativeToolchain>
```

Comment:
ok, I can do that as well in here. I just didn't know if UseDotNetNativeToolchain was used in a different place other than when running the tests, which is why I added that friendly error only when you actually want the tests to run.